src/api/transaction/suspended-payment-execution.js

```diff
@@ -18,40 +17,36 @@ type SuspendedPaymentExecution = {
 
 function createSuspendedPaymentExecutionTransaction(account: string,
       payment: SuspendedPaymentExecution
-): Transaction {
+): Object {
   validate.address(account);
   validate.suspendedPaymentExecution(payment);
 
-  const transaction = new Transaction();
-  transaction.suspendedPaymentFinish({
-    account: account,
-    owner: payment.owner,
-    paymentSequence: payment.paymentSequence
-  });
+  const txJSON: Object = {
+    TransactionType: 'SuspendedPaymentFinish',
+    Account: account,
+    Owner: payment.owner,
+    OfferSequence: payment.paymentSequence
+  };
 
   if (payment.method) {
```

Comment:
I don't think we should ever rely on coercion to determine whether to set a property. Although `Method` will [probably start at 1](https://github.com/ripple/rippled/blob/6700e44793ff479bd81aa8a1008649ef229e1ffa/src/ripple/app/tx/impl/SusPay.cpp#L93-L95) this seems sloppy. The [schema](https://github.com/ripple/ripple-lib/blob/98422e4153e53b8006097c8f9cebbfa9d5f831d4/src/api/common/schemas/suspended-payment-execution.json#L14) seems right